mlir/lib/Dialect/OpenMP/IR/OpenMPDialect.cpp

```diff
@@ -1672,8 +1672,9 @@ void TargetOp::build(OpBuilder &builder, OperationState &state,
   // TODO Store clauses in op: allocateVars, allocatorVars, inReductionVars,
   // inReductionByref, inReductionSyms.
   TargetOp::build(builder, state, /*allocate_vars=*/{}, /*allocator_vars=*/{},
-                  makeArrayAttr(ctx, clauses.dependKinds), clauses.dependVars,
-                  clauses.device, clauses.hasDeviceAddrVars, clauses.ifExpr,
+                  clauses.bare, makeArrayAttr(ctx, clauses.dependKinds),
+                  clauses.dependVars, clauses.device, clauses.hasDeviceAddrVars,
+                  clauses.ifExpr,
                   /*in_reduction_vars=*/{}, /*in_reduction_byref=*/nullptr,
                   /*in_reduction_syms=*/nullptr, clauses.isDevicePtrVars,
                   clauses.mapVars, clauses.nowait, clauses.privateVars,
```

Comment:
Nit: Looks like a formatting artifact.
```suggestion
                  clauses.ifExpr, /*in_reduction_vars=*/{},
                  /*in_reduction_byref=*/nullptr, /*in_reduction_syms=*/nullptr,
                  clauses.isDevicePtrVars, clauses.mapVars, clauses.nowait,
                  clauses.privateVars,
```